README.md

```diff
@@ -50,6 +51,14 @@ as in
 
     export TEST_NGINX_PORT=1989
 
+# INSTALLATION
+
+```bash
+perl Makefile.PL
+make
+sudo make install
+```
+
```

Comment:
BTW, this .md file is automatically generated from the POD docs in the file `lib/Test/Nginx.pm` by the [releng](https://github.com/agentzh/old-openresty/blob/master/bin/releng) utility.